versions.props

```
@@ -4,8 +4,8 @@ com.google.code.findbugs:jsr305 = 3.0.2
 com.google.guava:guava = 31.1-jre
 com.netflix.concurrency-limits:* = 0.2.2
 com.netflix.feign:feign-*= 8.18.0
-com.palantir.conjure.java:* = 7.4.0
-com.palantir.conjure.java:conjure-lib = 7.4.0
+com.palantir.conjure.java:* = 7.5.0
+om.palantir.conjure.java:conjure-lib = 7.5.0
```

sorry about the wild goose chase on this, something seems odd with `:conjure-java-client-verifier` requiring this pin, but fine leaving this extra pin here


```suggestion
com.palantir.conjure.java:conjure-lib = 7.5.0
```

---

👍 I'll follow up on this one separately

---

yeah, not urgent, just seemed odd when looking through pins